Xamarin.Forms.Core/IndicatorsView.cs

```diff
@@ -0,0 +1,285 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.ComponentModel;
+using System.Linq;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms
+{
+	public enum IndicatorShape
+	{
+		Circle,
+		Square
+	}
+
+	public class IndicatorsView : StackLayout
```

Comment:
there's no semantic reason for this to inherit from StackLayout. it would mean one could use the StackLayout API to add animated GIFs cats to the IndicatorView.

Inherit from View, and maintain an internal StackLayout.

or way better, inherit from TemplatedView

Reply:
agreed, i'll try